main.cpp

```diff
@@ -69,172 +69,173 @@ void variableDeclarations()
+{
     ignoreUnused(speedWind, rainHeight, sunLevel);
 }
-
-/*
- 9)
- */
+
+/*
+ 9)
+ */
 int computeHousePrice(int numberOfBedroom, int numberOfBathroom, int surface)
 {
     ignoreUnused(numberOfBedroom, numberOfBathroom, surface);
     return {};
-}
-/*
- 10)
- */
+}
+/*
+ 10)
+ */
 long caclulateFactorial(long number) 
 {
     ignoreUnused(number);
     return {};
-}
-/*
- MAKE SURE YOU ARE NOT ON THE MASTER BRANCH
-
- Commit your changes by clicking on the Source Control panel on the left, entering a message, and click [Commit and push].
- 
- If you didn't already: 
-    Make a pull request after you make your first commit
-    pin the pull request link and this repl.it link to our DM thread in a single message.
-
- send me a DM to review your pull request when the project is ready for review.
-
- Wait for my code review.
- */
-
-int main()
-{
-    //example of calling that function, storing the value, and passing it to ignoreUnused at the end of main()
-    auto carRented = rentACar(6, 2); 
-    
+}
+/*
+ MAKE SURE YOU ARE NOT ON THE MASTER BRANCH
+
+ Commit your changes by clicking on the Source Control panel on the left, entering a message, and click [Commit and push].
+ 
+ If you didn't already: 
+    Make a pull request after you make your first commit
+    pin the pull request link and this repl.it link to our DM thread in a single message.
+
+ send me a DM to review your pull request when the project is ready for review.
+
+ Wait for my code review.
+ */
+
+int main()
+{
+    //example of calling that function, storing the value, and passing it to ignoreUnused at the end of main()
+    auto carRented = rentACar(6, 2); 
+    
     //1)
-    auto calculate2Distance(25); 
-    
-    //2)
-    auto resistor =  calculateResistor(12.0, 5.2, true);
-    //3)
-    displayAge(25);
-    //4)
-    setSpeed(150, true);
-    //5)
-    displayFullName("Bob", "Sponge");
-    //6)
-    auto sum = computeSum(12, 2525245345);
-    //7)
-    auto time = caclulateTimeDilutionInMotion(152, 50);
-    //8)
-    forecastWeather(15, 25, 96);
-    //9)
-    auto price = computeHousePrice(5, 3, 250);
-    //10)
-    auto f = caclulateFactorial(125);
-    
+    auto calculate2Distance(25); 
+    
+    //2)
+    auto resistor =  calculateResistor(12.0, 5.2, true);
+    //3)
+    displayAge(25);
+    //4)
+    setSpeed(150, true);
+    //5)
+    printLetters('A', 'B');
+    //6)
+    auto sum = computeSum(12, 2525245345);
+    //7)
+    auto time = caclulateTimeDilutionInMotion(152, 50);
+    //8)
+    forecastWeather(15, 25, 96);
+    //9)
+    auto price = computeHousePrice(5, 3, 250);
+    //10)
+    auto f = caclulateFactorial(125);
+    
     ignoreUnused(
         carRented, 
         calculate2Distance, 
         resistor,
         displayAge, 
         setSpeed, 
-        displayFullName, 
+        printLetters, 
         sum, 
         time, 
         forecastWeather,
         price,
         f
```

Comment:
Don't pass functions to `ignoreUnused`.  only pass the local variables that are storing the values returned from the functions that return values.
- [ ] revise